### DISCUSSION TITLE
What other technical indicators you want this library to provide support for?

### DISCUSSION THREAD

**Discussion by cinar** [cinar/indicator#105] [Ideas]
What other technical indicators do you want this library to provide support for? I tried to add the ones that I am familiar with, but I will appreciate it if you can help me by providing ideas here.

**Comment by cinar** [comment]
@rene-post please let me know if this library is lacking any other indicators that you are using from other sources.

**Reply by rene-post** [reply]
Will do!

**Comment by vsemeniuk** [comment]
For VWAP, can you add support for standard deviation bands?
Perhaps we can define how many bands (1-4) and stdev. # per band.
